src/sky_plugin_swoole_curl.cc

```diff
@@ -79,7 +79,7 @@ void sky_plugin_swoole_curl(zend_execute_data *execute_data, const std::string &
 
             if (!Z_ISUNDEF_P(http_code) && Z_TYPE_P(http_code) == IS_LONG) {
                 span->addTag("status_code", std::to_string(Z_LVAL_P(http_code)));
-                if (Z_LVAL_P(http_code) >= 400) {
+                if (Z_LVAL_P(http_code) >= 400 || Z_LVAL_P(http_code) == 0) {
```

Comment:
Why ?

Reply:
When domian can't resovle, http_code is 0

Reply:
According to other SDK implementations, all achieve 0, I expect to be consistent